Backpack/NavigationBar/Classes/BPKNavigationBar.m

```diff
@@ -276,6 +276,7 @@ - (BPKNavigationBarButton *)leftButton {
     if (!_leftButton) {
         _leftButton = [[BPKNavigationBarButton alloc] init];
         _leftButton.translatesAutoresizingMaskIntoConstraints = NO;
+        _leftButton.title = @"";
```

Comment:
Kind of a hack, to remove margins on the sides when navigation bar buttons are icon only

Reply:
might that break other button usages where they somehow give it an empty string title? (particularly the hasTitle change)

Reply:
Actually no, this is just a default value for the title for the navigation bar buttons.
`hasTitle` will return `true` now, making the BPKButton think that it's not an icon only button. This makes the Link button to NOT have side margins, which is what's required for the navigation bar buttons.
This is a hack of course and a proper refactor on the Navigation Bar is needed, but it would require some ""big"" changes on the app